quarkus/service/src/main/java/org/apache/polaris/service/quarkus/config/ProductionReadinessChecks.java

```diff
@@ -151,7 +151,7 @@ public ProductionReadinessCheck checkTokenBrokers(
                   if (config
                       .tokenBroker()
                       .symmetricKey()
-                      .map(SymmetricKeyConfiguration::secret)
+                      .flatMap(SymmetricKeyConfiguration::secret)
```

Comment:
looking at https://github.com/apache/polaris/blob/e1c0a1ccdad4e4e432c026cdc3ae30dcb8112da1/service/common/src/main/java/org/apache/polaris/service/auth/AuthenticationRealmConfiguration.java#L73-L84
this might actually be a bug fix in that previously any `symmetricKey` was flagged as an error?
i.e. this line was checking `Optional.of(Optional.empty()).isPresent()` when the `SymmetricKeyConfiguration` is file-based.